src/agoradatatools/process.py

```diff
@@ -417,6 +413,15 @@ def process(
     upload: bool = upload_opt,
     auth_token: str = synapse_auth_opt,
 ):
+    """Process the configuration file and execute the data processing pipeline based on options.
+
+    Args:
```

Comment:
docstring added because this is marked missing by the pre-commit hook interrogate.

Reply:
Might as well go ahead and add the `None` return typehint too